### ISSUE TITLE
[a11y]:  Issue in Pagination component at 400% zoom

### ISSUE THREAD

**Issue by Vinodbs** [carbon-design-system/carbon#16200]
### Package

@carbon/react

### Browser

Chrome, Firefox

### Operating System

MacOS

### Package version

v1.50.0

### React version

v17.0.2

### Automated testing tool and ruleset

Manual

### Description

When Zoomed to 400% - in the Pagination Component ,the functionality of changing Items per page and Changing the Page number is not visible.

### Reproduction/example

https://stackblitz.com/edit/github-uvkdkp?file=src%2FApp.jsx

### Steps to reproduce

 - zoom the browser to 400%
 - Observe the issue - the functionality of changing Items per page and Changing the Page number is not visible.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/issues/16080#issuecomment-2029791810
